homework/index.js

```diff
@@ -30,18 +32,116 @@
     return elem;
   }
 
+  // create reusable function.
+  // to remove children of an html element which is (the parent).
+  function removeChildren(element) {
+    while (element.firstChild) {
+      element.removeChild(element.firstChild);
+    }
+  }
+
+  // create left box (repository Info).
+  function createRepoInfo(selectedRepo, repoContainer) {
+    const listRepoInfo = createAndAppend('ul', repoContainer, { class: 'repo-list' });
+
+    const li = createAndAppend('li', listRepoInfo, { text: 'Repository:  ', class: 'li' });
+
+    createAndAppend('a', li, {
+      target: '_blank',
+      href: selectedRepo.html_url,
+      text: selectedRepo.name,
+    });
+
+    createAndAppend('li', listRepoInfo, {
+      text: `Description: ${selectedRepo.description}`,
+      class: 'li',
+    });
+    createAndAppend('li', listRepoInfo, { text: `Forks: ${selectedRepo.forks}`, class: 'li' });
+    createAndAppend('li', listRepoInfo, {
+      text: `Updated: ${selectedRepo.updated_at}`,
+      class: 'li',
+    });
+  }
+
+  // create (right box) as a table to render contributor Info
+  function createContributor(selectedRepo, repoContainer, root) {
```

Comment:
You are handling **_all_** contributors here -> `createContributors` (mind the plural 's').

Reply:
: